vello_tests/tests/regression.rs

```diff
@@ -24,3 +25,22 @@ fn rounded_rectangle_watertight() {
         .unwrap()
         .assert_mean_less_than(0.001);
 }
+
+const DATA_IMAGE_PNG: &[u8] = include_bytes!("./data_image.png");
+
+#[test]
+#[should_panic(expected = "Expected mean to be less than 0.001, got 0.3072978")]
```

Comment:
I'm surprised by that expected number being so specific

Reply:
I just copy&pasted full panic msg because I forget that panic does not need to match fully.